src/chronometer.js

```diff
@@ -1,33 +1,62 @@
 class Chronometer {
   constructor() {
     // ... your code goes here
+      this.currentTime = 0
+      this.intervalId = null
   }
 
   start(printTimeCallback) {
     // ... your code goes here
+   this.intervalId = setInterval(() => {
+        this.currentTime++
+        if (printTimeCallback) printTimeCallback()
+    }, 1000);
   }
 
+
   getMinutes() {
     // ... your code goes here
+    if (this.currentTime === 0 ) {
+      return 0 
+    } else {
+      return Math.floor(this.currentTime / 60)
+    }
   }
 
   getSeconds() {
     // ... your code goes here
+    if (this.currentTime === 0) {
```

Comment:
Same thing here, no need for the condition just the `return this.currentTime % 60`